app/src/main/java/com/github/se/project/ui/lesson/AddLesson.kt

```diff
@@ -103,131 +96,165 @@ fun AddLessonScreen(
           { _, year, month, dayOfMonth ->
             val selectedCalendar = Calendar.getInstance()
             selectedCalendar.set(year, month, dayOfMonth)
-
-            if (selectedCalendar.before(currentDateTime)) {
-              Toast.makeText(context, "You cannot select a past date", Toast.LENGTH_SHORT).show()
-            } else {
-              selectedDate = "$dayOfMonth/${month + 1}/$year"
-            }
+            selectedDate = "$dayOfMonth/${month + 1}/$year"
           },
           calendar.get(Calendar.YEAR),
           calendar.get(Calendar.MONTH),
           calendar.get(Calendar.DAY_OF_MONTH))
-
-  // Restrict date picker to future dates only
   datePickerDialog.datePicker.minDate = currentDateTime.timeInMillis
 
   // Time Picker
   val timePickerDialog =
       TimePickerDialog(
```

Comment:
Is it wanted that the timePickerDialog is light blue? Since all the others screens are purple it is a bit suprising (but not important at all and also okay if it was wanted)
Here is what I got when running the MainActivity: 
![photo_2024-10-17_19-27-14](https://github.com/user-attachments/assets/cf6738ef-eb75-499c-8740-bc482184fbbf)

Reply:
I will fix this